### PR TITLE
feat: home page added

### PR DESCRIPTION
**Pull request by humzahasan**
Added home page basic layout

This will be the base layout and more enhancement to be made soon. Not the final version yet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,64 +4,90 @@
...
+          <div class="home-installation-left">
+            <h3 class="mg-title">
+              In order to start using our component library, you need to include
+              the said lines in the head tag of your html page.
+            </h3>
+          </div>
+          <div class="home-installation-right">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=38px&ph=21px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Ffeedyui.netlify.app%252Flibrary%252Flib.css%2522%2520%252F%253E"
+              style="
```
Comment (sach10-create) [inline index.html:70]: Just a suggestion : avoid inline css

**Review by bhushannaikaroskar — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (humzahasan) [reply index.html:70]: Good catch. I just started working on the documentation site. Will be adding common style for all iframe's.

**Review by sach10-create — APPROVED** [review]
Nice naming conventions.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.home-navitem.home-navitem-active {
+  border-bottom: 1px solid var(--primary-color);
+}
+/* 
+.home-hero > img {
+  height: 20rem;
+  width: 20rem;
+} */
```
Comment (iRohitGaur) [inline style.css:48]: remove commented code

**Review by iRohitGaur — COMMENTED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**